### PR TITLE
Create 2021-03-18.md

### PR DESCRIPTION
**Pull request by felixfaassen**
add meeting notes product-steering 18th march 2021

### CODE REVIEW

**Comment by felixfaassen** [comment]
@someren Here are the meeting notes from our last product steering meeting from Signalen. Could you review them so I can merge them into our product steering repository? Thanks and have a great weekend!

File: meeting-notes/2021-03-18.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+tags: Signalen, Product Steering group
+---
+
+# Signalen Product Steering Meeting 18th of March, 2021, 11:00 to 12:00
+
+Chair: Amy van Someren
```
Comment (someren) [inline meeting-notes/2021-03-18.md:7]: Maybe we should add which municipalities were present? 's-Hertogenbosch, Utrecht and halfway Almere joined, Amsterdam was absent as was Urk?

**Comment by someren** [comment]
@felixfaassen did you receive my review? I added one remark, rest is approved. I checked the box 'viewed'. Can you take it from here?

File: meeting-notes/2021-03-18.md (hunk 1 above)
Comment (felixfaassen) [reply meeting-notes/2021-03-18.md:7]: Hi @someren great idea! I will add that.

**Comment by felixfaassen** [comment]
> @felixfaassen did you receive my review? I added one remark, rest is approved. I checked the box 'viewed'. Can you take it from here?

Hi @someren yes I did ;-) thanks

**Pull request merged**